src/comlink.ts

```diff
@@ -45,7 +45,9 @@ export type Remote<T> =
     T extends { new (...args: infer R1): infer R2 }
       ? { new (...args: R1): Promise<Remote<R2>> }
       : unknown
-  );
+  ) & {
+    releaseProxy: () => Promise<void>;
```

Comment:
I am worried about this potentially clashing. I think it’d be preferable if we exported a symbol from Comlink that you can use to access this function similar to `createEndpoint`

Reply:
fixed.